core/common/modules/connect/module.php

```diff
@@ -199,12 +199,12 @@ public function get_subscription_plans( $context = '' ) {
 			],
 			static::ACCESS_LEVEL_PRO => [
 				'label' => 'Pro',
-				'promotion_url' => "https://elementor.com/pro/?utm_source={$context}&utm_medium=wp-dash&utm_campaign=gopro",
+				'promotion_url' => "https://elementor.com/pro/?utm_source=$context&utm_medium=wp-dash&utm_campaign=gopro",
```

Comment:
Second thought on this - the variable can easily be mistaken for a part of the string here.
So for nobody to do a mistake here again, maybe it is better to do:
`'https://elementor.com/pro/?utm_source=' . $context . '&utm_medium=wp-dash&utm_campaign=gopro'`